PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -660,6 +660,16 @@ public final class DefaultControlsViewController: ContentControlsViewController
             showBrandedContentButton.layer.cornerRadius = showBrandedContentButton.frame.height / 2
             showBrandedContentButton.setTitle(currentUIProps.brandedContentTitle,
                                               for: .normal)
+            if nextUIProps.isBrandedContentClickable {
+                let image = UIImage(named: "open-web-page", in: Bundle(for: type(of: self)), compatibleWith: nil)!
```

Comment:
Could you use just `UIImage(named: "open-web-page")`

Reply:
Probably not because the image is not located in `mainBundle`.